packages/location_permissions/android/src/main/java/com/baseflow/location_permissions/LocationPermissionsPlugin.java

```diff
@@ -60,52 +67,45 @@
 
   @Retention(RetentionPolicy.SOURCE)
   @IntDef({
-    SERVICE_STATUS_DISABLED,
-    SERVICE_STATUS_ENABLED,
-    SERVICE_STATUS_NOT_APPLICABLE,
-    SERVICE_STATUS_UNKNOWN,
+          SERVICE_STATUS_DISABLED,
+          SERVICE_STATUS_ENABLED,
+          SERVICE_STATUS_NOT_APPLICABLE,
+          SERVICE_STATUS_UNKNOWN,
   })
   private @interface ServiceStatus {}
 
-  private final Registrar mRegistrar;
+  private Context context;
```

Comment:
Please rename this to `applicationContext` since it will contain the Android application context and naming it as such is makes it much clearer what we are dealing with.